### DISCUSSION TITLE
Made a python script to download all the attachments from the generated json.

### DISCUSSION THREAD

**Discussion by Nojyto** [Tyrrrz/DiscordChatExporter#906] [General]
Did not find a way to download all the attachments from a chat with this method. So I made a python script and I thought I should share it. https://github.com/Nojyto/discordFileDownloader

**Comment by 96-LB** [comment]
Does this provide any benefits over the download media (`--media` on CLI) option?

**Reply by Nojyto** [reply]
Threading
Renames all the files

**Reply by Nojyto** [reply]
To be honest I didn't know there was a download media option :d

**Reply by 96-LB** [reply]
It's accessible on the GUI under the advanced export settings pop-up you get after you choose which channels to export
